src/iop/watermark.c

```diff
@@ -311,8 +370,73 @@ static gchar *_watermark_get_svgdoc(dt_iop_module_t *self, dt_iop_watermark_data
       svgdata = svgdoc;
     }
 
-    // Current image ID
+    // Simple text from watermark module
     gchar buffer[1024];
+
+    if (p->font)
```

Comment:
Completely bogus check since p->font is not a pointer but an array.
This broke clang build.